app/models/concerns/journeys/sessions/pii_attributes.rb

```diff
@@ -0,0 +1,28 @@
+module Journeys
+  module Sessions
+    module PiiAttributes
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :pii_attributes, default: [], instance_writer: false
+      end
+
+      class_methods do
+        def attribute(name, type = nil, **options)
+          pii_attributes << name if options.delete(:pii)
+          super
+        end
+      end
+
+      def attributes_with_pii_redacted
```

Comment:
can we add some basic coverage for this method?

Reply:
Done 👍